### ISSUE TITLE
Add Broadcast TX to Explorer UI

### ISSUE THREAD

**Issue by christroutner** [Permissionless-Software-Foundation/jwt-bch-frontend#49]
## Overview
This task depends on [this previous task](https://github.com/Permissionless-Software-Foundation/bch-api/issues/6) being completed first.

The scope of this task is to update the Explorer UI for [mainnet](https://fullstack.cash/explore-mainnet) and [testnet](https://fullstack.cash/explore-testnet/) with a new ElectrumX endpoint for the new 'broadcast tx' endpoint completed in the previous task.

## Specification

- Update the [mainnet JSON file](https://github.com/Permissionless-Software-Foundation/jwt-bch-frontend/blob/master/src/data/mainnet-routes.json) to add an entry for the new endpoint.
- Update the [testnet JSON file](https://github.com/Permissionless-Software-Foundation/jwt-bch-frontend/blob/master/src/data/testnet-routes.json) to add an entry for the new endpoint.
- Both UIs should include entries for the POST call. (No GET call)

## Acceptance Criteria
The changes will be evaluated in the browser. If the changes result in a successful operation, the PR will be merged into the `master` branch. Upon merging of the PR, the bounty will be paid out.

**Comment by christroutner** [comment]
Sorry to take so long to review this.

When I click the 'Execute' button, I get this error:

```
{
  "success": false,
  "error": "request body must be a string."
}
```

We had a previous discussion about how the content-type was `text` and not `JSON`. I think when you made these changes, they were to an older version of bch-api. I updated this endpoint in bch-api to access a JSON object with `txHex` property containing the hex:

https://github.com/Permissionless-Software-Foundation/bch-api/blob/master/src/routes/v3/electrumx.js#L541

Can you update this PR to reflect these changes? The bch-api changes are live on the production server now.

**Comment by christroutner** [comment]
Issue resolved by PR #58. 150 PSF tokens rewarded. TXID: 4ecc17f847df50036c74212cae9336e4bd2d369b6ef566e8f7f20caded17ac02
